cmd/mcpserver/storage.go

```diff
@@ -0,0 +1,48 @@
+package mcpserver
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/kubescape/storage/pkg/generated/clientset/versioned"
+	spdxv1beta1 "github.com/kubescape/storage/pkg/generated/clientset/versioned/typed/softwarecomposition/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func CreateKsObjectConnection(namespace string, maxElapsedTime time.Duration) (spdxv1beta1.SpdxV1beta1Interface, error) {
+	var cfg *rest.Config
```

Comment:
We have code that is doing a similar initialization in the httphandler (https://github.com/kubescape/kubescape/blob/b3251306d034055c48af3fe1192e8867e445230c/httphandler/main.go#L63C6-L63C23)

Reply:
Refactoring